contents/blog/guardrail-metrics.md

```diff
@@ -0,0 +1,103 @@
+
+![](../images/blog/guardrail-metrics/trigger.png)
+
+## What guardrail metrics should you pick?
+
+As a general guideline, you should pick metrics important to the entire product or company. These are often [north star](/blog/north-star-metrics) or [product health](/blog/product-health-metrics) metrics. Specifically, Airbnb breaks guardrail metrics into three categories
+
+- Key business metrics
+- User experience metrics
+- Strategic priority metrics
+
+It is important to have a mix of metrics from each category to ensure a broad enough coverage of potential issues. It is also important to not have too many metrics. Adding one increases the likelihood of false positives, which limits the throughput of experiments. For example, an [A/A test](/tutorials/aa-testing) with a significance level of 0.05 would expect to cause the following amount of false positives:
+
+- With 3 guardrail metrics, you would have at least one false alert approximately 14% of the time.
+- With 10 guardrail metrics, approximately 40% of the time.
+- With 25 guardrail metrics, approximately 73% of the time.
+
+### Example guardrail metrics
+
+For more insight into what specific metrics you should try, you can look at what other companies use as guardrail metrics. 
+
+- Airbnb: revenue, bounce rate, page load speed, seats booked for experiences.
+
+- [Square](https://developer.squareup.com/blog/lessons-learned-from-running-web-experiments/): bottom of funnel conversion, page load speed
+
+- [Bing](https://exp-platform.com/Documents/2017-08%20KDDMetricInterpretationPitfalls.pdf): page load time, page returns
+
+- [Netflix](https://www.adventuresinwhy.com/pdf/beyond_ab_testing.pdf): sample mismatch ratio, engagement, retention, conversion.
+
+Other potential guardrail metrics include clickthrough rate, active users, customer lifetime value, and pageviews. 
+
+> **Tip:** To help pick a metric, **i**t is useful to know what secondary metrics you have stopped experiments for in the past. These are the organization’s [revealed preference](https://en.wikipedia.org/wiki/Revealed_preference) for guardrail metrics and likely make good choices to formalize.
+
+## Your step-by-step guide to implementing guardrail metrics
+
+Now that you know all about guardrail metrics, it’s time to implement them in your A/B tests. 
+
+1. Create an experiment in PostHog.
+
+2. Fill out the key name, variants, goal metric, and any other details. Adding details about the guardrail metrics to the description is useful.
+
+3. Under "secondary metrics," click "Add metric," and set up the trend or funnel you want as a guardrail. 
+
+4. Click "Save," implement the experiment in your app, and press "Launch."
+
+5. Guardrail metrics display alongside your goal metrics on the experiment detail screen.
```

Comment:
Would be nice to have a screenshot here, maybe showing where to see the secondary metrics being tracked?